templates/Register.html

```diff
@@ -23,22 +26,114 @@
         <input name="spotifytoken" id="spotify" type="checkbox">
     </p>
 
+
     <Label for="pin">4 character pin: </Label>
     <input name="pin" type="text" Placeholder="****" required pattern=".{4,4}" maxlength="4" minlength="4">
     <br>
     <p>Picture of face:</p>
-    <input name="facepath"  type="file" accept="image/*" capture="camera" required>
+    <input name="facepath" id="facepath"  type="file" accept="image/*" capture="camera" required>
```

Comment:
You do not need to ask for an image or path. Josh and I will be writing the script to get the facial recognition stuff working and we will save it directly to the DB.

Reply:
In the write ups we did. The front end was supposed to pull an image from the phone(camera/storage) or device they were using on the front end. This prompts for a camera/ file that can be stored.

Reply:
The thing is, the api we will be using for this is in Python, so it makes more sense to just have the mirror take the picture. Besides there is no way to activate the camera on the phone from a web page

Reply:
The script there enables capture of images with the a phone. I haven't been able to test it yet, I'll take another look online after class. At one of the meeting we talked about not having a physical button that the mirror uses and just getting the image through the registration page. If I misunderstood that is fine but I'm pretty sure our documentation shows us getting it on the registration page.

Reply:
It lets you upload an image but to capture one.

it should be triggered after we send the preferences and pin to the back end. The facial recognition script should start up and then save the image.

Reply:
Alright I can change that in the next commit. I won't be back home for awhile so I can't currently change it.